src/main/java/wolox/training/models/Book.java

```diff
@@ -124,63 +126,70 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
-        this.author = author;
+        this.author = Preconditions.checkNotNull(author,"Author must not be null");
     }
 
     public String getImage() {
         return image;
     }
 
     public void setImage(String image) {
-        this.image = image;
```

Comment:
It is good to do the check before assigning it, doing the check on the variable, and not on the attribute.